### ISSUE TITLE
kiosk: deleting site as identifier in edit file dialogue leads to incorrect modified field

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2935]
Issue found in testing the first bullet point in #2807 

Added the context LA to image 5245b723-d45b-416f-bd2a-632089af4031 at around 8:20ish Pacific Time, 28 X 2024. The modified field does not reflect that addition: 

<img width="430" alt="Screenshot 2024-10-28 at 8 26 36 PM" src="https://github.com/user-attachments/assets/5cc8a4a2-fc4c-4f27-81c0-0325f5b84f69">

**Comment by luizaogs** [comment]
Same issue for deletion. Deleted a context from image f2c07043-8b8b-4dd8-9b0c-3159910e0c7c around 8:28pm Pacific Time and the modified field does not reflect that: 

<img width="298" alt="Screenshot 2024-10-28 at 8 30 00 PM" src="https://github.com/user-attachments/assets/576d80d6-e93f-4968-b510-177d17890850">

**Comment by luizaogs** [comment]
Um, weirder - don't know if you want a separate ticket. 

Tried deleting site PVD from image 86c7ad59-27a9-4425-9f51-91b7662c7ee8 and that DID get the modified field changed. But it changed incorrectly, to Eastern time when it should be Pacific time (as reflected in the timezone under my user icon in the Hub). BUT it also didn't delete the identifier. If I click out of the file edit dialog and back in, PVD is still there (though the modified field has changed, even if incorrectly).

<img width="389" alt="Screenshot 2024-10-28 at 8 33 00 PM" src="https://github.com/user-attachments/assets/2a1de579-5e28-4861-b700-1fdd1165d4e6">

**Comment by urapadmin** [comment]
ja, it tripped me up, too, again: Adding / Deleting a context must not change the image's own timestamp. That would be wrong. 
The time stamp of adding it would be reflected in the record that connects the image to the context. So in LA dayplans. In Q&V in the dayplans or in the recording app you should see the correct modification ts. 

That's why my list in #2807 is misleading. Deleting really does not do anything at all and modifying is deleting and then adding, so the same as adding.

**Comment by urapadmin** [comment]
> Um, weirder - don't know if you want a separate ticket.
> 
> Tried deleting site PVD from image 86c7ad59-27a9-4425-9f51-91b7662c7ee8 and that DID get the modified field changed. But it changed incorrectly, to Eastern time when it should be Pacific time (as reflected in the timezone under my user icon in the Hub). BUT it also didn't delete the identifier. If I click out of the file edit dialog and back in, PVD is still there (though the modified field has changed, even if incorrectly).
> <img alt="Screenshot 2024-10-28 at 8 33 00 PM" width="389" src="https://private-user-images.githubusercontent.com/67970402/380961534-2a1de579-5e28-4861-b700-1fdd1165d4e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAxNzM4NDAsIm5iZiI6MTczMDE3MzU0MCwicGF0aCI6Ii82Nzk3MDQwMi8zODA5NjE1MzQtMmExZGU1NzktNWUyOC00ODYxLWI3MDAtMWZkZDExNjVkNGU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI5VDAzNDU0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWViYTg3MDgwMzA2ZjdiM2Y0ZDkyMGY1YjU2NDY1ZmU4NDUzYTYzYTFjNjgwNzM2MmE3NGNkMGE5MWU1YWEwOGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tRzRkNa9h_i_RCFh5faspfpf-PSTkz0is46wD1rbgAs">

well, ja, that is weird on two fronts: You cannot delete an image from site or add an image to site (it would need #616 to change that). That's an oooold issue I have never gotten to. So the first issue here is that I am allowing something that I shouldn't (but I'd rather tackle #616 than change that).

Why the modification date changes, though, I don't know. That is indeed an issue then.

**Comment by luizaogs** [comment]
Right then, I guess I’ll change the name of the ticket? Or do you want a new one?

**Comment by urapadmin** [comment]
new name would be great. This must be a very site map specific issue.
